theme6/source/partials/_footer.erb

```diff
@@ -0,0 +1,40 @@
+<div class="contenedor">
+      <footer>         
+          <div class="contenedorsocial">
+            <div class="box-social">
+              <p class="socialmedia" i>FOLLOW US</p>
+              <div class="boxsocialmedia">
```

Comment:
How come you didn't use dashes here like you did in `box-social`. This class can be renamed like this `box-social-media` or `box-socialmedia`. Whichever is fine.